sources/us/tx/lubbock.json

```diff
@@ -5,16 +5,16 @@
         "state": "tx",
         "county": "Lubbock"
     },
-    "data": "http://216.167.160.18:88/2014/rest/services/2015External/MapServer/0",
-    "type": "ESRI",
+    "data": "http://ewebmap.ci.lubbock.tx.us/currentdata/Parcels/ParcelOwnerLanduseAddress.zip",
+    "type": "zip",
```

Comment:
This should be `http` and the `conform` object's `type` should be `shapefile-polygon`.